src/server/modes/charts/plugins/datalens/preparers/treemap/d3.ts

```diff
@@ -0,0 +1,262 @@
+import type {
```

Comment:
Almost a complete copy of the data preparation for highcharts chart except for the difference in data format and things that didn't make sense